app/controllers/api/v1/netflix_shows_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Api::V1::NetflixShowsController < ApplicationController
+  def show
+    render json: { netflix_show: NetflixShow.find(params[:id]) }
```

Comment:
Lets instead do 
```ruby
render json:  NetflixShow.find(params[:id])
```
This will work more easily with serializers when we implement them in the future